scripts/Player/Player.gd

```diff
@@ -0,0 +1,30 @@
+extends KinematicBody
+
+
+export var _move_speed := 30.0
+export(float, 0, 2) var mouse_sensitivity := 1.0
+
+onready var camera := $Camera
+
+var direction_vector := Vector3.ZERO
+Area
+
+func _physics_process(delta: float) -> void:
+	var forward_movement := Vector3.ZERO
```

Comment:
I don't really get what (forward_movement and sideways_movement) will be used for

Reply:
They will be used to move player character forward/backward and left/right. It's separated for clarity instead of just movement_vector. They will be added together and normalized if you wish to move player in two directions simultaneously (front/left/ back/right etc)

Reply:
good point